builds/misc/packages-release.yaml

```diff
@@ -0,0 +1,1116 @@
+- job: Job_9
+  displayName: Ubuntu 20.04 - amd64
+  pool:
+    name: Azure-IoT-Edge-1ES-Hosted-Linux
+    demands:
+    - ImageOverride -equals agent-aziotedge-ubuntu-18.04-docker
+  steps:
+  - checkout: self
+    submodules: recursive
+  - task: Bash@3
+    displayName: Set Environment
+    inputs:
+      targetType: inline
+      filePath: echo "##vso[task.setvariable variable=RUSTUP_HOME;]$VSTS_WORK/rustup" echo "##vso[task.setvariable variable=CARGO_HOME;]$VSTS_WORK/cargo" echo "##vso[task.setvariable variable=PATH;]$VSTS_WORK/cargo/bin:$PATH"  echo '##vso[task.setvariable variable=PACKAGE_OS;]ubuntu18.04' echo '##vso[task.setvariable variable=PACKAGE_ARCH;]amd64'
+      script: >-
+        echo "##vso[task.setvariable variable=RUSTUP_HOME;]$VSTS_WORK/rustup"
+
+        echo "##vso[task.setvariable variable=CARGO_HOME;]$VSTS_WORK/cargo"
+
+        echo "##vso[task.setvariable variable=PATH;]$VSTS_WORK/cargo/bin:$PATH"
+
+
+        echo '##vso[task.setvariable variable=PACKAGE_OS;]ubuntu20.04'
+
+        echo '##vso[task.setvariable variable=PACKAGE_ARCH;]amd64'
+  - task: ShellScript@2
+    displayName: Create libiothsm and iotedged packages
+    inputs:
+      scriptPath: edgelet/build/linux/package.sh
+  - task: CopyFiles@2
+    displayName: Copy libiothsm Files to Artifact Staging
+    inputs:
+      SourceFolder: edgelet/target/hsm
+      Contents: '*.deb'
+      TargetFolder: $(build.artifactstagingdirectory)
+  - task: CopyFiles@2
+    displayName: Copy iotedged Files to Artifact Staging
+    inputs:
+      SourceFolder: edgelet/target/release
+      Contents: '*.deb'
+      TargetFolder: $(build.artifactstagingdirectory)
+  - task: ManifestGeneratorTask@0
+    displayName: SBOM Generation Task
+    inputs:
+      BuildDropPath: $(build.artifactstagingdirectory)
+  - task: PublishBuildArtifacts@1
+    displayName: Publish Artifacts
+    condition: succeededOrFailed()
+    inputs:
+      PathtoPublish: $(build.artifactstagingdirectory)
+      ArtifactName: iotedged-ubuntu20.04-amd64
+      TargetPath: '\\my\share\$(Build.DefinitionName)\$(Build.BuildNumber)'
+- job: Job_10
+  displayName: 'Test: Install Ubuntu 20.04 amd64 packages'
+  dependsOn: Job_9
+  pool:
+    name: Azure-IoT-Edge-1ES-Hosted-Linux
+    demands:
+    - ImageOverride -equals agent-aziotedge-ubuntu-20.04-docker
+  steps:
+  - checkout: self
+    submodules: recursive
+  - task: DownloadBuildArtifacts@0
+    displayName: Download Build Artifacts
+    inputs:
+      artifactName: iotedged-ubuntu20.04-amd64
+      extractTars: false
+  - task: Bash@3
+    displayName: Install libiothsm-std and iotedge
+    inputs:
+      targetType: inline
+      script: >-
+        # Ensure docker group is created
+
+        if ! getent group iotedge >/dev/null; then
+            addgroup --system docker
+        fi
+
+
+        sudo docker --version
+
+        sudo apt-get --yes install $(System.ArtifactsDirectory)/iotedged-ubuntu20.04-amd64/*.deb
+- job: Job_11
```

Comment:
Also add Ubuntu 20.04 arm32v7? This would be published to GitHub release page, but not to packages.microsoft.com. Not a hard requirement, but good for completeness, since we already publish Ubuntu 18.04 arm32v7...

Reply:
Done!